docs/products/opensearch/howto/upgrade-to-opensearch-with-terraform.rst

```diff
@@ -78,6 +78,14 @@ Bring the Terraform state back in sync with your OpenSearch service by importing
 
     terraform import 'aiven_opensearch.os' <project-name>/<service-name>
 
+Describe and import dependent resources if exist.
```

Comment:
I would move this to the end of the block and use the following working: 
If you have had any Elasticsearch ALCs and users, do not forget to import Opensearch counterparts to the Terraform state.